test/unit/spec/v1/MfaVerify_spec.js

```diff
@@ -2232,22 +2238,17 @@ Expect.describe('MFA Verify', function() {
         })
         .then(function(test) {
           expect(test.form.hasErrors()).toBe(true);
-          expect(test.form.errorMessage()).toBe('Password is incorrect');
```

Comment:
Where did this message come from previously?

Reply:
Previously this message came from `errorCauses[0].errorSummary` in mock response
https://github.com/okta/okta-signin-widget/blob/9d0b5302faa8eac667197cc20598be96c6711336/test/unit/helpers/xhr/MFA_VERIFY_invalid_password.js#L11
because of this line
https://github.com/okta/okta-signin-widget/blob/e63951ffcff0fd6843199d546c52ee332db93c0a/src/util/Util.js#L98